### PR TITLE
added S11 size

### PR DESCRIPTION
**Pull request by revathi-kogentix**
Adding changes

### CODE REVIEW

File: templates/azure/cloudera/azuredeploy.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
       "defaultValue": "S5",
       "allowedValues": [
         "S5",
-        "S9", 
+        "S9",
+        "S11"'
```
Comment (AjayDec) [inline templates/azure/cloudera/azuredeploy.json]: Typo comma vs quote

File: templates/azure/cloudera/dhcp/data-node-linear-s11.json [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+        }
+      }
+    }
+  ],
+    "outputs": {
+        "ipsComputed": {
+            "type": "string",
+            "value": "[concat(reference(concat(variables('nicName'),'0')).ipConfigurations[0].properties.privateIPAddress,',',  reference(concat(variables('nicName'),'1')).ipConfigurations[0].properties.privateIPAddress,',', reference(concat(variables('nicName'),'2')).ipConfigurations[0].properties.privateIPAddress,',', reference(concat(variables('nicName'),'3')).ipConfigurations[0].properties.privateIPAddress,',', reference(concat(variables('nicName'),'4')).ipConfigurations[0].properties.privateIPAddress,',', reference(concat(variables('nicName'),'5')).ipConfigurations[0].properties.privateIPAddress,',', reference(concat(variables('nicName'),'6')).ipConfigurations[0].properties.privateIPAddress,',', reference(concat(variables('nicName'),'7')).ipConfigurations[0].properties.privateIPAddress,',', reference(concat(variables('nicName'),'9')).ipConfigurations[0].properties.privateIPAddress)]"
+        }
```
Comment (AjayDec) [inline templates/azure/cloudera/dhcp/data-node-linear-s11.json:341]: closing brackets for concat

**Review by AjayDec — CHANGES_REQUESTED** [review]

**Comment by revathi-kogentix** [comment]
Fixed now.

**Comment by revathi-kogentix** [comment]
Fixed now.

**Pull request closed without merging**